aspnet/mvc/views/working-with-forms.rst

```diff
@@ -0,0 +1,621 @@
+- HTML Helper alternative: `@Html.ValidationMessageFor <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/Rendering/HtmlHelperValidationExtensions/index.html>`__ 
+
+The `Validation Message Tag Helper <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/TagHelpers/ValidationMessageTagHelper/index.html>`__  is used with the ``asp-validation-for`` attribute on a HTML `span <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span>`__ element.
+
+.. code-block:: HTML
+  
+  <span asp-validation-for="Email">
+  
+The Validation Message Tag Helper will generate the following HTML:
+
+.. code-block:: HTML
+
+    <span class="field-validation-valid" 
+      data-valmsg-for="Email" 
+      data-valmsg-replace="true">
+
+You generally use the `Validation Message Tag Helper <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/TagHelpers/ValidationMessageTagHelper/index.html>`__  after an ``Input`` Tag Helper for the same property. Doing so displays any validation error messages near the input that caused the error.
+
+:Note: You must have a view with the correct JavaScript and `jQuery <https://jquery.com/>`__ 
+ script references in place for client side validation. See :doc:`/mvc/models/validation` for more information.
+ 
+When a server side validation error occurs (for example when you have custom server side validation or client-side validation is disabled), MVC places that error message as the body of the ``<span>`` element.
+
+.. code-block:: HTML
+
+   <span class="field-validation-error" data-valmsg-for="Email" 
+               data-valmsg-replace="true">
+      The Email Address field is required.
+   </span>
+ 
+The Validation Summary Tag Helper
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Targets ``<div>`` elements with the ``asp-validation-summary`` attribute 
+- HTML Helper alternative: ``@Html.ValidationSummary``
+
+The `Validation Summary Tag Helper <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/TagHelpers/ValidationSummaryTagHelper/index.html>`__  is used to display a summary of validation messages. The ``asp-validation-summary`` attribute value can be any of the following:
+
++-------------------------------+--------------------------------+
+|asp-validation-summary         |  Validation messages displayed |  
++===============================+================================+
+|ValidationSummary.All          | Property and model level       |
++-------------------------------+--------------------------------+  
+|ValidationSummary.ModelOnly    | Model                          |
++-------------------------------+--------------------------------+  
+|ValidationSummary.None         | None                           |
++-------------------------------+--------------------------------+  
+
+If the form has validation errors, the browser reports the validation errors and the form data is not posted to the server.
```

Comment:
Remove this sentence. The point is covered in the _Validation_ document and doesn't belong in this one. In any case it has nothing to do w/ either validation tag helper: Form submission will be prevented even if none of the client-side errors are displayed (in HTML or tag helpers).